washingtonpost/elex-clarity
# Feat: add option for race name maps + cleanup

## Description

This PR adds a lookup object so we can make the race names consistent from county to county in Georgia. More detail in the comments on this PR: https://github.com/washingtonpost/elex-clarity/pull/2

Looking for some general feedback on this, but one main question I have is how we want to structure these lookup objects. Right now I'm just using state postal -- but I think something using state postal and the Clarity election ID might be even better? It's unclear how we're going to use this on an ongoing basis (are inconsistent race names just a Georgia thing?)...

## Jira Ticket

None

## Test Steps

I've added a new fixture for Gwinnett County, Georgia, which has combo English/Spanish race names. The unit tests I added should cover it. You can also run `elexclarity 105369 GA --level=precinct --filename="tests/fixtures/gwinnett_precincts_11-3.xml" > gwinnett.json` and check the names in the output to make sure they match the expected result.
